packages/input/test/lion-input.test.js

```diff
@@ -1,8 +1,8 @@
 import { Validator } from '@lion/form-core';
-import { expect, fixture, triggerFocusFor, aTimeout } from '@open-wc/testing';
-import { html, unsafeStatic } from 'lit/static-html.js';
-import { getInputMembers } from '../test-helpers/index.js';
 import '@lion/input/define';
+import { aTimeout, expect, fixture, html, triggerFocusFor, unsafeStatic } from '@open-wc/testing';
+import {} from 'lit/static-html.js';
```

Comment:
?

Reply:
Only this line needs to be removed. Apart from that it looks great.

Maybe also flatten the commits?
- Update to Lit v2.0.2 (on abdonrd's name)
- fix: use html & unsafeStatic from @open-wc/testing (on your name)